<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>-30</y>
      <width>1291</width>
      <height>901</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pokemon Hollow</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(92, 132, 255, 255), stop:1 rgba(255, 128, 128, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="steps">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>931</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 127);
font: 24pt &quot;Pokemon Solid&quot;;</string>
    </property>
    <property name="text">
     <string>P! Seeect Pokemon </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Pkm1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>PKm1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Pkm2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>240</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 85, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Pkm2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Type1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);

font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="Status1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pokemon Solid</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 24pt &quot;Pokemon Solid&quot;;</string>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="Level1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);

font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Level</string>
    </property>
   </widget>
   <widget class="QLabel" name="Level2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>320</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 85, 127);

font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Level</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Type2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>400</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 85, 127);

font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Status2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>620</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 85, 127);

font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="total2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>150</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(236, 13, 13);
font: 75 36pt &quot;Calibri&quot;;
border-color: rgb(85, 85, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="total1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(236, 13, 13);
font: 75 36pt &quot;Calibri&quot;;
border-color: rgb(85, 85, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="P1TM">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(170, 170, 255);
color: rgb(85, 170, 127);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string>TM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="P2Atk1">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>480</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);

font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 170, 127);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string>ATK1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="P2Atk2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>550</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(85, 170, 255);
color: rgb(85, 170, 127);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string>ATk2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="P2TM">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>620</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(133, 0, 200);

font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 170, 127);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string>TM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="P1Atk1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);

font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 170, 127);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string>ATK1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="P1Atk2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 127);
color: rgb(85, 170, 127);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string>Atk2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="P1Atk1_B">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>+2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="P1Atk2_B">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>550</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>+2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="P1TM_B">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>630</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>+2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="P2Atk1_B">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>490</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 255, 127);</string>
    </property>
    <property name="text">
     <string>+2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="P2Atk2_B">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>560</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 255, 127);</string>
    </property>
    <property name="text">
     <string>+2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="P2TM_B">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>630</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 255, 127);</string>
    </property>
    <property name="text">
     <string>+2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Effect1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pokemon Solid</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 24pt &quot;Pokemon Solid&quot;;</string>
    </property>
    <property name="text">
     <string>effect</string>
    </property>
   </widget>
   <widget class="QLabel" name="Effect2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>560</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 85, 127);

font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Effect</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Extra1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);

font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Extra</string>
    </property>
   </widget>
   <widget class="QLabel" name="Extra2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>320</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 85, 127);

font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Extra</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="AtkSelected1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pokemon Solid</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 24pt &quot;Pokemon Solid&quot;;</string>
    </property>
    <property name="text">
     <string>Atk Selected</string>
    </property>
   </widget>
   <widget class="QLabel" name="AtkSelected2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>470</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pokemon Solid</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 85, 127);
font: 24pt &quot;Pokemon Solid&quot;;</string>
    </property>
    <property name="text">
     <string>Atk Selected</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="AtkPower1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pokemon Solid</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 24pt &quot;Pokemon Solid&quot;;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="AtkPower2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>470</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pokemon Solid</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 85, 127);
font: 24pt &quot;Pokemon Solid&quot;;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
